src/NetAnalyzers/Core/Microsoft.NetCore.Analyzers/Runtime/DisposeObjectsBeforeLosingScope.cs

```diff
@@ -141,13 +141,6 @@ public override void Initialize(AnalysisContext context)
                                 return;
                             }
 
-                            if (disposeAnalysisResult.ControlFlowGraph.OriginalOperation.HasAnyOperationDescendant(o => o.Kind == OperationKind.None && !(o.Parent is INameOfOperation)))
```

Comment:
Can we add a test for nameof case or already exists from when this workaround was added? I think it is the latter, but just confirming.

Reply:
There is already a test (see #3309) but thanks for making me check!